README.md

```diff
@@ -21,10 +21,9 @@ Basic usage of this module is as follows:
 ```hcl
 module "folders" {
   source  = "terraform-google-modules/folders/google"
-  version = "~> 0.1"
+  version = "> 1.0.0"
```

Comment:
We should always recommend pinning to a minor version. Pinning to a major version risks pulling in breaking changes unexpectedly.

Reply:
Make sense, updated